root/usr/bin/discovery-fetch-extensions

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Use a commandline option, fdi.zips=zip1.zip,zip2.zip,etc and
+# get the zips from the TFTP server. Try zipserver passed on kernel
+# cmdline first and fall back to DHCP next-server passed in through
+# ENV
+
+local extdir="/opt/extension"
```

Comment:
Sorry; transparent proxy gave me an outdated fdi-image.tar during download to my test system.